public/content/lessons/2017/eulers-formula-via-group-theory/index.mdx

```diff
@@ -1,9 +1,215 @@
+
+<Figure image="add_multiply_homo.png" width="60%" />
+
+It’s weird and strange to think about a function that takes in one kind of action and spits out another kind of action, but that’s actually something which comes up all the time in group theory. This exponential property is very important for this association between groups, it forms a map between the additive and multiplicative group for real numbers.
+
+This exponentiation function guarantees that if I compose two sliding actions, maybe a slide by $-1$, then a slide by $2$, it corresponds to composing the output actions, in this case squishing by $2^{-1}$, then stretching by $2^2$.
+$$
+\begin{align*}
+2^{-1+2}&=2^{-1}\cdot2^2 \\
+\exp(-1+2)&=\exp(-1)\cdot\exp(2)
+\end{align*}

+$$
+Mathematicians would describe a property like this by saying the function “preserves the group structure.” In this sense, the arithmetic within a group is what gives it its structure, and function like this exponential plays nicely with that arithmetic. Functions between groups that preserve the arithmetic like this are really important throughout group theory, enough so that they’ve earned themselves a fancy name: “homomorphisms.”
+
+<Figure image="homomorphism.png" width="60%" />
+
+## Complex Exponentiation
+
+Now think about what this means for associating the additive group in the complex plane with the multiplicative group in the complex plane. We already know that when you plug in a real number to $2^x$, you get out a real number, a positive real number in fact, so this exponential function takes any pure horizontal slide and turns it into some pure stretching or squishing action.
+
+<Figure image="real_exponent.png" video="real_exponent.mp4" />
+
+So it would be reasonable for this new dimension of additive actions, slides up and down, to map to this new dimension of multiplicative actions, pure rotations.
+
+<Figure image="imag_exponent.png" video="imag_exponent.mp4" />
+
+Each vertical sliding action corresponds to some point on the vertical axis of imaginary numbers. Using the exponentiation function, each point gets mapped to a point on the unit circle corresponding to a multiplicative rotating action.
+
+<Figure image="imaginary_axis.png" video="imaginary_axis.mp4" />
+
+In fact, for the function $2^x$, the input $i$, a vertical slide of 1 unit, happens to map to a rotation of $0.693$ radians. A walk around the unit circle covering $0.693$ units of distance. With a different exponential function, like $5^x$, that input $i$, a vertical slide of 1 unit, maps to a rotation of $1.609$ radians; a walk around the unit circle covering exactly $1.609$ units of distance.
+
+<Figure image="2x5x.png" width="60%" />

```

Comment:
I would consider adding the animations for this example and some of the following.